src/Symfony/Bundle/FrameworkBundle/Controller/RedirectController.php

```diff
@@ -176,4 +186,49 @@ public function __invoke(Request $request): Response
 
         throw new \RuntimeException(sprintf('The parameter "path" or "route" is required to configure the redirect action in "%s" routing configuration.', $request->attributes->get('_route')));
     }
+
+    private static function parseQueryString(string $qs)
```

Comment:
this implementation should be replaced by #37272 when merging up to master (it's exactly the same)